src/main/java/br/ufpe/cin/mergers/handlers/RenamingConflictsHandler.java

```diff
@@ -26,14 +28,15 @@
 
     public static void handle(MergeContext context) {
         //when both developers rename the same method/constructor
```

Comment:
Yes, only for methods and constructors. Which other declarations you suggest? Field declarations would only be relevant in case of multiple lines field declarations, otherwise unstructured and semistructured merge report the same conflicts (we have talked about this long time ago). Entire files are handled reporting the same result of unstructured merge, also discussed long time ago. Finally, innerclasses are handled bye other handler (DeletionsHandler).

Reply:
@guilhermejccavalcanti what about files with more than one class?

Reply:
@gaabs since this handler is focused on methods and constructors renaming and deletion, can you update its name? I accept suggestions for the name, as it will be huge.

Reply:
OperationRenamingConflictHandler

Reply:
Since it handles deletions too, I would pick ```MethodAndConstructorRenamingAndDeletionHandler```
It is necessary to update the command line parameter @jvcoutinho